framework/src/eventloop/eventloop_internal.h

```diff
@@ -0,0 +1,26 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __EVENTLOOP_INTERNAL_H__
+#define __EVENTLOOP_INTERNAL_H__
+
+/* Wrapper of allocation APIs */
+#define EL_ALLOC(a)  malloc(a)
```

Comment:
for using malloc and free, stdlib.h is needed.

Reply:
Done